components.yaml

```diff
@@ -37,6 +37,7 @@ GEOS_Util:
   local: ./src/Shared/@GMAO_Shared/@GEOS_Util
   remote: ../GEOS_Util.git
   tag: v2.0.3
+  sparse: ./config/GEOS_Util.sparse
```

Comment:
The config file might need to be named more specifically for GEOSldas, something like `GEOS_Util_ldas.sparse`  (taking my cues from the config file for GEOSgcm_GridComp)

Reply:
This sparse config file belongs to GEOSldas, so "ldas" probably is not necessary